### ISSUE TITLE
A11y: Underline links in the pages

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/vacant-lots-proj#268]
## Description

Hyperlinks shouldn't use color alone to convey they are links. Let's underline them as well

<details>
<summary> Screenshot</summary>

<img width="826" alt="Screenshot 2024-01-24 at 01 13 43" src="https://github.com/CodeForPhilly/vacant-lots-proj/assets/102739972/c88a92ac-9402-4137-b1e1-280b2d57096f">

</details>

## Acceptance Criteria
-  Links found in the following pages should be underlined:
  - https://www.cleanandgreenphilly.org/transform-property
  - https://www.cleanandgreenphilly.org/methodology
  - https://www.cleanandgreenphilly.org/about 

## Additional context
Touches the following WCAG 2.1 Criteria
[Success Criteria 1.4.1 Use of Color (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/use-of-color.html)
